bgpd/bgpd.c

```diff
@@ -7509,6 +7509,34 @@ int peer_ttl_security_hops_unset(struct peer *peer)
 	return ret;
 }
 
+static void peer_reset_message_stats(struct peer *peer)
+{
```

Comment:
Can a peer be NULL, btw?

Reply:
The vty peer and peer-group lookup functions return NULL if the neighbor/group ID can not be found. The current code path catches this, so I could remove that check. I put it there just in case someone adds other calls to this functions in the future.